tools/disass.py

```diff
@@ -0,0 +1,795 @@
+#!/usr/bin/env python3
+
+import argparse
+import subprocess
+import pygments
+
+from bisect import bisect
+from collections import namedtuple, defaultdict, UserString, UserList
+
+from pygments import lexers
+from pygments import formatters
+
+from pyparsing import (Word, Char, Literal, Opt, ZeroOrMore, OneOrMore, Regex,
+                       Suppress, Combine, White, Group)
+from pyparsing.exceptions import ParseException
+from blessings import Terminal
```

Comment:
`blessing` isn't installed by default. It'd be a good idea to perhaps add this to `requirements.txt` or some kind of `requirements-dev.txt`

Reply:
Done!